setup-node/action.yml

```
@@ -51,8 +48,8 @@ runs:
       working-directory: ${{ inputs.working-directory }}
 
     - name: Authenticate with Github Packages
-      if: ${{ inputs.install-dependencies == 'true' && inputs.github_npm_access_token }}
-      run: echo "//npm.pkg.github.com/:_authToken=${{ inputs.github_npm_access_token }}" > ~/.npmrc && echo "@buoysoftware:registry=https://npm.pkg.github.com/" >> .npmrc
+      if: ${{ inputs.install-dependencies == 'true' && inputs.github_npm_access_token == '' }}
```

Should we remove the reference to the input?
```suggestion
      if: ${{ inputs.install-dependencies == 'true' }}
```

---

Ah! Good catch this should actually be:`inputs.github_npm_access_token == ''`